agent/pkg/api/socket_routes.go

```diff
@@ -111,19 +111,35 @@ func websocketHandler(w http.ResponseWriter, r *http.Request, eventHandlers Even
 	}
 
 	for {
-		_, msg, err := ws.ReadMessage()
-		if err != nil {
-			if _, ok := err.(*websocket.CloseError); ok {
-				logger.Log.Debugf("Received websocket close message, socket id: %d", socketId)
-			} else {
-				logger.Log.Errorf("Error reading message, socket id: %d, error: %v", socketId, err)
+		// params[0]: query
+		// params[1]: enableFullEntries (0: disable, 1: enable)
+		params := make([][]byte, 2)
+		breakWholeLoop := false
+		for i := range params {
+			_, params[i], err = ws.ReadMessage()
+			if err != nil {
+				if _, ok := err.(*websocket.CloseError); ok {
+					logger.Log.Debugf("Received websocket close message, socket id: %d", socketId)
+				} else {
+					logger.Log.Errorf("Error reading message, socket id: %d, error: %v", socketId, err)
+				}
+
+				breakWholeLoop = true
+				break
 			}
+		}
 
+		if breakWholeLoop {
 			break
 		}
 
+		enableFullEntries := false
+		if len(params[1]) > 0 && params[1][0] != 48 {
```

Comment:
48 is key code for "0"?
maybe need to explain this consts somehow?

Reply:
With https://github.com/up9inc/mizu/pull/819/commits/df4405eedd399abc7ac50b72ebff5bcfce610f12 I have changed the criteria to being empty or non-empty.